wayland-egl-test/wayland-egl.c

```diff
@@ -1,3 +1,6 @@
+// The contents of this file is free and unencumbered software released into the
```

Comment:
Didn't we agree to add the whole license text?

See also https://unlicense.org/
> For the time being you'll probably want to retain the whole shebang.

If so, then please simply copy files from my repository https://github.com/dwrobel/tutorials where this is already added.

Reply:
done